packages/expo-ads-admob/android/src/main/java/expo/modules/ads/admob/AdMobInterstitialAdModule.java

```diff
@@ -136,6 +137,15 @@ public void run() {
   }
 
   private void recreateInterstitialAdWithAdUnitID(String adUnitID) {
+    if (mActivityProvider.getCurrentActivity() == null) {
```

Comment:
Since `mActivityProvider.getCurrentActivity()` will be used twice in this function, it would be better to get it only once, store as a variable and reuse later where `InterstitialAd` is initialized.

Reply:
Done.